src/frontend/src/btc/components/send/BtcSendForm.svelte

```diff
@@ -18,6 +20,13 @@
 	let amountError: BtcAmountAssertionError | undefined;
 	let invalidDestination: boolean;
 
+	let invalid = true;
```

Comment:
This will grows with more condition?

It's exactly the same rule as in IcSendForm, so just wondering if we should add a TODO to potentially refactor it some day.

Reply:
Yes, in case of BTC this flag will have more conditions, but still it would be nice to align some basic validation checks with other network's SendForms. Added TODO comment.